design/user_data_incentives.md

```diff
@@ -0,0 +1,119 @@
+# User interaction with Obscuro 
+
+In a typical transparrent blockchain, there are service providers like "Infura" who act as the interface to the blockchain from the point of view of most users.
```

Comment:
Typo `transparrent -> transparent'

Worth noting on the Infura side that they are benefitting from data. Being able to see all user request and transaction traffic is probably a treasure-trove for market analysis firms, block building firms etc. that I'm sure they sell on.

The obscuro nodes won't get this data benefit from servicing encrypted requests so it's an additional reason to need incentives (as well as the extra cost you mention).